src/PKSim.Presentation/Services/BuildingBlockTask.cs

```diff
@@ -89,6 +89,18 @@ public void Edit(IPKSimBuildingBlock buildingBlockToEdit)
          }
       }
 
+      private ICloneBuildingBlockPresenter getCloneBuildingBlockPresenter(IPKSimBuildingBlock buildingBlockToClone)
```

Comment:
One centralized place for the cloen so that no matter where it's being called from, we get the right presneter